### PR TITLE
chore: add princerajpoot20 as code owner for ds and ui

### PR DESCRIPTION
**Pull request by Amzani**
I suggest @princerajpoot20 as a code owner for the design system `apps/design-system/` and ui `packages/ui/` as he did major contribution to build the new components, and he is committed to support new studio vision and plans #634.

### Some contributions
- https://github.com/asyncapi/studio/pull/708
- https://github.com/asyncapi/studio/pull/710
- https://github.com/asyncapi/studio/pull/711
- https://github.com/asyncapi/studio/pull/718
- https://github.com/asyncapi/studio/pull/792
- https://github.com/asyncapi/studio/pull/808

In addition to committing to this repos he is also reviewing the work of others (e.g https://github.com/asyncapi/studio/pull/784 / https://github.com/asyncapi/studio/pull/773)

### CODE REVIEW

**Review by fmvilas — APPROVED** [review]

**Comment by fmvilas** [comment]
Absolutely! 👏

**Comment by princerajpoot20** [comment]
Thank you so much for proposing me.🙇🏻‍♂️🙇🏻‍♂️ I am fully committed and will give my best. 🙂

**Review by princerajpoot20 — COMMENTED** [review]
I am not sure, but I guess `asyncapi-bot-eve` should also be added here.

File: CODEOWNERS
Comment (princerajpoot20) [inline CODEOWNERS:12]:
```suggestion
apps/design-system/ @fmvilas @mcturco @KhudaDad414 @Amzani @princerajpoot20 @asyncapi-bot-eve
packages/ui/ @fmvilas @mcturco @KhudaDad414 @Amzani @princerajpoot20 @asyncapi-bot-eve
```

**Review by fmvilas — CHANGES_REQUESTED** [review]
@princerajpoot20 is right. The bot has to be there, I think.

**Review by KhudaDad414 — APPROVED** [review]
@fmvilas @princerajpoot20 but the `@asyncapi-bot-eve` is already there with full code ownership. am I missing something? 🤔 

btw, welcome aboard @princerajpoot20 ㊗️ 🥳

**Comment by princerajpoot20** [comment]
> @fmvilas @princerajpoot20 but the @asyncapi-bot-eve is already there with full code ownership. am I missing something? 🤔

@KhudaDad414 If the changes are `limited` to specific areas, such as `DS or UI`, then only the `code owners of those areas` are called for review. If the changes **include any files outside of these DS/UI segments**, then full code owners are called.

**Comment by KhudaDad414** [comment]
@princerajpoot20 the point of `asyncapi-bot-eve` is to approve bot PRs. not sure if it cares about being `called for review`. it should approve other bot PRs anyways. 🤔

**Comment by fmvilas** [comment]
True. It already has full code ownership 👍

**Review by fmvilas — APPROVED** [review]

**Comment by princerajpoot20** [comment]
@KhudaDad414 Yes agree, it is there to approve bot PR only.

**Comment by Amzani** [comment]
Welcome @princerajpoot20 😊🎉

**Comment by fmvilas** [comment]
Welcome, @princerajpoot20! 🎉 

So now that you're a code owner, you have the right to join the AsyncAPI Technical Steering Committee. If you want to join, please submit a PR updating the [MAINTAINERS.yaml](https://github.com/asyncapi/community/blob/master/MAINTAINERS.yaml). Example: https://github.com/asyncapi/community/pull/801.

**Comment by princerajpoot20** [comment]
@fmvilas @Amzani @KhudaDad414 

I wanted to say a **big thank you**🙏. Thank you for **trusting** me and having **faith** in me. **This means a lot to me.** ❤️

I am truly grateful and honored to be part of this awesome community. 🙇‍♂️ 

Your support means **a lot to me.** In this journey, I have learned and grown a lot. I will give **my best** 💯 and do **every possible thing** I can for the betterment of the **Studio** and for this **community.** 🙇‍♂️

**Comment by princerajpoot20** [comment]
> So now that you're a code owner, you have the right to join the AsyncAPI Technical Steering Committee. If you want to join, please submit a PR updating the [MAINTAINERS.yaml](https://github.com/asyncapi/community/blob/master/MAINTAINERS.yaml).

Definitely, I would love to join TSC, and be a part of this awesome community. 🚀 ❤️

**Pull request merged**
